ruby_review_week_four.rb

```diff
@@ -1,20 +1,41 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+# Something that can store data.
+# Example:
+# my_var = 30
 
 # 2. What Ruby method can you use to make a loop from an array?
+# The .each method.
+# Example:
+# numbers = [ 1,2,3,4,5]
+
+# numbers.each do |number|
+# 	puts number
+# end
 
 # 3. In Ruby, what are booleans?
+# Things that are either true or false.
+# boolean == true
 
 # 4. How do classes relate to objects?
+# Objects are created and belong to a certain class.
 
 # 5. What is an instance variable?
+# I think it's just a normal variable inside a method.
```

Comment:
an instance variable is an `attribute` of an object, so if i have have a dog object that has an instance variable of `@name` then the value of the name is an attribute of the dog